src/solvers/refinement/axt.h

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************\
+
+ Module: Expression Building Utility
+
+ Author: Diffblue Limited. All rights reserved.
+
+\*******************************************************************/
+#ifndef CPROVER_SOLVERS_REFINEMENT_AXT_H
+#define CPROVER_SOLVERS_REFINEMENT_AXT_H
+
+#include <util/expr.h>
+#include <util/string_expr.h>
+
```

Comment:
I would have thought this would be happier in util/ as nothing it does is specific to refinement.  Also, maybe we should just resurrect oexpr.